src/PIL/PdfParser.py

```diff
@@ -825,8 +825,9 @@ def get_value(cls, data, offset, expect_indirect=None, max_nesting=-1):
                 try:
                     stream_len = int(result[b"Length"])
                 except (TypeError, KeyError, ValueError) as e:
-                    msg = "bad or missing Length in stream dict (%r)" % result.get(
-                        b"Length", None
+                    msg = (
+                        "bad or missing Length in stream dict "
+                        f"({result.get(b'Length')})"
                     )
```

Comment:
```suggestion
                    stream_len = result.get(b"Length")
                    msg = f"bad or missing Length in stream dict ({stream_len})"
```
https://github.com/radarhere/Pillow/pull/25 feels that error message should fit onto one line. The change above this would be my suggestion for that goal.

The suggestion from the PR is
```python
stream_len_str = result.get(b"Length")
try:
	stream_len = int(stream_len_str)
except (TypeError, KeyError, ValueError) as e:
	msg = f"bad or missing Length in stream dict ({stream_len_str})"
	raise PdfFormatError(msg) from e
```

I think that makes the code harder to read, because, if you ignore the exception, there's more to understand.

Reply:
My suggestion would be: https://github.com/radarhere/Pillow/pull/25#discussion_r1592841987

Reply:
Ok, I've pushed that - minus the `KeyError`, because now that `result[b"Length"]` is gone, it will no longer happen. `int(None)` will instead raise a TypeError.